examples/generate_dataset.py

```diff
@@ -0,0 +1,28 @@
+
```

Comment:
Following the comment below, to follow the structure that we already have, could you please add a quickstart notebook? similar to https://github.com/Microsoft/Recommenders/blob/staging/notebooks/00_quick_start/sar_single_node_movielens.ipynb

Reply:
Maybe you don't need this file. In the notebooks, we don't use synthetic data, but public datasets such as Movielens. In this case we could make a notebook with Movielens or Criteo.

Reply:
I will remove this script to avoid too much workload on notebook...